data/cache/test/test_cache.py

```diff
@@ -13,6 +13,15 @@
 
 DATA = {}
 
+CACHE_CONFIG = {
```

Comment:
Can you also set the previous default values in `config.py`

Reply:
Can you add a `TEST_` prefix so that it's easier to differentiate.
We should also add mock values for other fields that are expected in the config: `engine`, `endpoint`, ...

Reply:
👍